lib/rhc/commands/deployment.rb

```diff
@@ -74,7 +74,9 @@ def activate(id)
 
       raise RHC::DeploymentsNotSupportedException.new if !rest_app.supports? "DEPLOY"
 
-      activate_deployment(rest_app, id)
+      ssh_executable = check_ssh_executable! options.ssh
+
+      activate_deployment(rest_app, id, ssh_executable, !!options.ssh)
```

Comment:
`!` in ruby (and in most dynamic languages) will return the opposite boolean value of the argument. In ruby, `nil` and `false` are the only things that evaluate to false. Since `options.ssh` will only ever either be `nil` or a string, `!!options.ssh` essentially is the same as `!options.ssh.nil?`, just shorter.